graylog2-web-interface/public/stylesheets/graylog2.less

```diff
@@ -1312,6 +1312,21 @@ td.limited {
   padding-top: 1px;
 }
 
+.ldap-test-login-result {
```

Comment:
Done! For now I put it in the same level as the component, as I think it's easier to see when files move or disappear, but I'm open for feedback on that.